src/wasmopcodecnt.c

```diff
@@ -38,6 +38,13 @@ static const char* s_infile;
 static const char* s_outfile;
 static size_t s_cutoff = 0;
 static const char* s_separator = ": ";
+static WasmBool s_report_opcode_dist = WASM_TRUE;
+static WasmBool s_report_i32_const_dist = WASM_TRUE;
+static WasmBool s_report_i32_load_dist = WASM_TRUE;
+static WasmBool s_report_i32_store_dist = WASM_TRUE;
+static WasmBool s_report_get_local_dist = WASM_TRUE;
+static WasmBool s_report_set_local_dist = WASM_TRUE;
+static WasmBool s_report_tee_local_dist = WASM_TRUE;
```

Comment:
Would it make sense use a static WasmOpcntData here rather than duplicating its members like this?